CIME/XML/env_mach_specific.py

```diff
@@ -231,6 +231,7 @@ def make_env_mach_specific_file(self, shell, case, output_dir=""):
         case: case object
         output_dir: string - path to output directory (if empty string, uses current directory)
         """
+        source_cmd = "." if shell == "sh" else "source"
```

Comment:
Why this if test?  "." is the standard so just use that.

Reply:
I tried using `.` in `csh` and got the error `/usr/bin/.: Permission denied.`, so I checked the `csh` man page and only saw references to using `source` in `csh`.

Reply:
@rljacob I tried this on my local ubuntu machine and anvil.